<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teamselection</class>
 <widget class="QDialog" name="teamselection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>211</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Players</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_delete">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="player_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_number">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
  </widget>
  <widget class="QListWidget" name="list_team">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_close">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>23</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_load_file">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_store_file">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>73</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Store</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_import_file">
   <property name="geometry">
    <rect>
     <x>124</x>
     <y>72</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
